include/http/router.js

```diff
@@ -55,62 +56,62 @@ module.exports = function (pb) {
         }
 
         static replaceMiddleware(name, middleware) {
-            let index = this._indexOfMiddleware(name);
-            if(index >= 0 && middleware) {
-                this.internalMiddlewareStack[index] = middleware;
+                let index = this._indexOfMiddleware(name);
+                if (index >= 0 && middleware) {
+                    this.internalMiddlewareStack[index] = middleware;
+                }
```

Comment:
MInor: it seems there is an issue with the indentation of these lines

Reply:
Now my eyes cant stop looking to see it.  I think its there.... maybe...,.